src/bindings/Vscode.ml

```diff
@@ -480,8 +469,21 @@ module LanguageClient = struct
     -> t = "LanguageClient"
     [@@bs.new] [@@bs.module "vscode-languageclient"]
 
+  external start : t -> unit = "start" [@@bs.send]
+
+  external stop : t -> unit = "stop" [@@bs.send]
+
   external onReady : t -> unit Promise.t = "onReady" [@@bs.send]
 
+  external sendRequest :
+       t
+    -> string
```

Comment:
Let's introduce a label for the string argument and polymorphic arguments. `~meth` and `~data` to match the names in the LSP spec